src/Parser/DemonHunter/Havoc/Modules/Abilities.js

```diff
@@ -44,17 +51,7 @@ class Abilities extends CoreAbilities {
           suggestion: true,
           recommendedEfficiency: 0.95,
           extraSuggestion: `This plus Nemesis and Metamorphosis make up your huge windows.`,
-        },
-      },
-      {
-        spell: SPELLS.MOMENTUM_TALENT,
```

Comment:
Did momentum fall out of favor?  Why remove it entirely?

Reply:
Momentum is a buff you gain from casting fel rush and vengeful retreat. It is not an ability you cast. It should never have been in abilities in the first place.
http://www.wowhead.com/spell=206476/momentum

Reply:
Gotcha, I know nothing about Havoc